Tests/test_imagefontctl.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+from helper import unittest, PillowTestCase
+from PIL import Image
+from PIL import ImageDraw
+
+FONT_SIZE = 20
+FONT_PATH = "Tests/fonts/DejaVuSans.ttf"
+
+try:
+    from PIL import ImageFont
+
+    # check if raqm is available
+    ttf = ImageFont.truetype(FONT_PATH, FONT_SIZE)
```

Comment:
There should be a way to check if raqm is installed. As it stands, if there is an error that triggers an import error on ImageFont, then this whole set of tests are going to get skipped.

There are version flags for several of the libraries, that would be something to add to _imagingft.c, potentially for all three libraries, but at least raqm.